wd_helper.au3

```diff
@@ -729,34 +728,27 @@ Func _WD_FrameEnter($sSession, $vIdentifier)
 	ElseIf IsInt($vIdentifier) Then
 		$sOption = '{"id":' & $vIdentifier & '}'
 	Else
-		_WinAPI_GUIDFromString("{" & $vIdentifier & "}")
-		If @error Then
-			$iErr = $_WD_ERROR_InvalidArgue
-		Else
-			$sOption = '{"id":' & __WD_JsonElement($vIdentifier) & '}'
-		EndIf
+		$sOption = '{"id":' & __WD_JsonElement($vIdentifier) & '}'
```

Comment:
I would like to suggest to use this way:
https://github.com/Danp2/au3WebDriver/issues/474#issuecomment-1554168745

Reply:
I disagree. See my reply https://github.com/Danp2/au3WebDriver/issues/474#issuecomment-1554476512 for my reasoning.

Reply:
Can be closed after my ASAP check...